server/hardware.c

```diff
@@ -612,6 +652,18 @@ void hw_close_table(struct hw_table *table)
 }
 
 
+size_t hw_get_queued_words(struct hw_table *table, unsigned int number)
+{
+    if (table->table_type == LONG_TABLE)
+    {
+        size_t result = 0;
+        ioctl(table->long_table.block_ids[number], PANDA_BLOCK_NWORDS, &result);
```

Comment:
Probably ought to check return from `ioctl` even if we just log the result (I presume the calling context for this function is unable to handle an error?)  There's a pattern for this, something like
```c
ERROR_REPORT(TEST_IO(ioctl(...)));
```

Reply:
There is no reason for this ioctl to fail, it is only copying one variable from kernel space to user space.